contractcourt/chain_arbitrator.go

```diff
@@ -1056,13 +1056,46 @@ type forceCloseReq struct {
 	closeTx chan *wire.MsgTx
 }
 
+// forceCloseContractOption is used for providing functional options to callers
+// of ForceCloseContract(). We don't export this type directly, but rather the
+// concrete functions implementing this type, so that callers can choose the
+// concrete function that best describes their use case for force closing.
+type forceCloseContractOption func(*ChannelArbitrator)
```

Comment:
Hmm, this doesn't look like the normal option pattern, as it's just a function type. Maybe just call it `forceCloseContractCallback` instead?